.plzconfig.bootstrap

```diff
@@ -1,13 +1,2 @@
-; Used during initial bootstrap, mostly to get rules loaded.
```

Comment:
I think maybe you can totally remove this file (and the references to `--profile  bootstrap`, can you give that a go?